runners/portability/test_pipeline_jar.sh

```diff
@@ -28,6 +28,16 @@ case $key in
         shift # past argument
         shift # past value
         ;;
+    --spark_job_server_jar)
```

Comment:
Can we just call it `JOB_SERVER_JAR`?

Reply:
Sure.

Reply:
Actually... I remember the reason I kept the flink_ and spark_ prefixes was so that the gradle build script would match the actual python args.

Reply:
I changed it.